README.md

```diff
@@ -110,6 +110,57 @@ without updating/overwriting any existing data from the target model.
 }}
 ```
 
+#### Incremental overwrite on hive models
+
+In case that the target incremental model is being accessed with 
+[hive](https://trino.io/docs/current/connector/hive.html) Trino connector,  an `insert overwrite` 
+functionality can be achieved when using:
+
+```
+hive.insert-existing-partitions-behavior=OVERWRITE
```

Comment:
Perhaps worth to mention that `hive` here is a catalog name?? In the example profile below we have `minio`, it could be misleading.